gerrit-entrypoint.sh

```diff
@@ -168,7 +168,13 @@ if [ "$1" = "/gerrit-start.sh" ]; then
   [ -z "${HTTPD_LISTENURL}" ] || set_gerrit_config httpd.listenUrl "${HTTPD_LISTENURL}"
 
   #Section gitweb
-  set_gerrit_config gitweb.cgi "/usr/share/gitweb/gitweb.cgi"
+  if [ "${GITWEB_TYPE}" = 'gitiles' ]  ; then
```

Comment:
Yes I tested all change I've done.
If you want to have a further look just see Google Git repo where Gerrit is hosted:
https://code.googlesource.com
The reasons to for using gitiles are: Nicer UI, support for Markdown